Source/WebKit/UIProcess/AuxiliaryProcessProxy.h

```diff
@@ -214,10 +214,10 @@ class AuxiliaryProcessProxy
 #endif
 
     enum ResumeReason : bool { ForegroundActivity, BackgroundActivity };
-    virtual void sendPrepareToSuspend(IsSuspensionImminent, double remainingRunTime, CompletionHandler<void()>&&) = 0;
-    virtual void sendProcessDidResume(ResumeReason) = 0;
+    virtual void sendPrepareToSuspend(IsSuspensionImminent, double remainingRunTime, CompletionHandler<void()>&&) { };
+    virtual void sendProcessDidResume(ResumeReason) { };
     virtual void didChangeThrottleState(ProcessThrottleState) { };
-    virtual ASCIILiteral clientName() const = 0;
+    virtual ASCIILiteral clientName() const { return nullptr; }
```

Comment:
Why is this change needed?

Reply:
If you did this as a fix for your earlier crashes, I do not believe this is correct. It is not OK to call virtual functions in constructors and destructors. It doesn't matter if pure virtual or not.

Reply:
Ah, good point! I will fix in the upcoming patch.

Thanks for reviewing!